### ISSUE TITLE
wgengine/monitor: evt=[unexpected] on Windows

### ISSUE THREAD

**Issue by bradfitz** [tailscale/tailscale#980]
```
2020-12-01 17:53:50.3404124 -0500 -0500: monitor: got windows change event after 231.7395ms: evt=route-o
2020-12-01 17:53:50.3664085 -0500 -0500: monitor: got windows change event after 31.721ms: evt=[unexpected] addr.internal=0; route.internal=0
``` 

/cc @alexbrainman

**Comment by alexbrainman** [comment]
Reading the code

https://github.com/tailscale/tailscale/blob/be6fe393c5b43e055e2c4b5601d0d89822852aba/wgengine/monitor/monitor_windows.go#L194-L200

it appears both oaddr.Internal and oroute.Internal are set to _STATUS_WAIT_0.

I don't see anything strange here - both address and route changed at the same time. No? It is just our code does not handle this situation gracefully.

Maybe I will spend some time simplifying this code.

Alex

**Comment by bradfitz** [comment]
This was fixed by 258d0e8d9a50aa04cd5503d5493eef20cfedfbce
